jobs/config.json

```diff
@@ -117,6 +130,13 @@
   ]
 },
 
+"ci-kubernetes-node-kubelet-1.6": {
+  "scenario": "kubernetes_kubelet",
+  "args": [
+    "--branch=release-1.6"
```

Comment:
it's frustrating that I have to put the branch in both the jenkins config and this file.